From 08ae19613c985f79b760659e79986b03e5a91364 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Wed, 26 Jan 2022 15:17:23 +0200
Subject: [PATCH 45/50] plat/nxp: s32g: Change FIP offset for QSPI and SD boot

A new set of values for FIP_QSPI_OFFSET and FIP_MMC_OFFSET
are needed after eliminating the gap between APP header and APP code.

Issue: ALB-8431
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32_common.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/plat/nxp/s32/s32_common.mk b/plat/nxp/s32/s32_common.mk
index 953f807fb..ae59836c8 100644
--- a/plat/nxp/s32/s32_common.mk
+++ b/plat/nxp/s32/s32_common.mk
@@ -185,14 +185,14 @@ ${BL2_W_DTB}: bl2 dtbs
 	@dd if=${BUILD_PLAT}/bl2.bin of=$@ bs=1024 seek=8 status=none
 
 # User defined parameters, for example:
-# 	make FIP_MMC_OFFSET=0x5400 <...other parameters>
+# 	make FIP_MMC_OFFSET=0x5240 <...other parameters>
 # These defines update only BL2's view of FIP AppBootCode:Code position.
 # IVT header updates (e.g. mkimage application code offset) should be updated
 # independently
 # These offsets must be aligned to the block size of 512 bytes
-FIP_MMC_OFFSET		?= 0x3400
+FIP_MMC_OFFSET		?= 0x3240
 $(eval $(call add_define,FIP_MMC_OFFSET))
-FIP_QSPI_OFFSET		?= 0x3400
+FIP_QSPI_OFFSET		?= 0x2240
 $(eval $(call add_define,FIP_QSPI_OFFSET))
 
 # If FIP_MEM_OFFSET is defined, the FIP is not read from boot source (QSPI/MMC)
-- 
2.17.1

